protocol/messenger.go

```diff
@@ -3433,6 +3434,17 @@ func (m *Messenger) handleRetrievedMessages(chatWithMessages map[transport.Filte
 							continue
 						}
 
+					case protobuf.NodeStatusUpdate:
+						p := msg.ParsedMessage.Interface().(protobuf.NodeStatusUpdate)
+						m.outputToCSV(msg.TransportMessage.Timestamp, msg.ID, senderID, filter.Topic, filter.ChatID, msg.Type, p)
+						logger.Debug("Handling NodeStatusUpdate", zap.Any("message", p))
+						err = m.HandleNodeStatusUpdate(messageState, p)
```

Comment:
This doesn't seem to take into account that this message has not originated from the community owner.
Even if this is just a "Hey I'm online" ping, theoretically any node can send these for any community. 

So if we want to keep this, we should probably check whether we're interested in this particular node status update in the first place, before we process it.